rt_utils/image_helper.py

```diff
@@ -165,20 +207,21 @@ def get_slice_contour_data(series_slice: Dataset, contour_sequence: Sequence):
     return slice_contour_data
 
 
-def get_slice_mask_from_slice_contour_data(series_slice: Dataset, slice_contour_data):
+def get_slice_mask_from_slice_contour_data(series_slice: Dataset, slice_contour_data,
+        transformation_matrix: np.ndarray):
     slice_mask = create_empty_slice_mask(series_slice)
-    for contour_coords in slice_contour_data:    
-        fill_mask = get_contour_fill_mask(series_slice, contour_coords)
+    for contour_coords in slice_contour_data:
+        fill_mask = get_contour_fill_mask(series_slice, contour_coords, transformation_matrix)
         # Invert values in the region to be filled. This will create holes where needed if contours are stacked on top of each other
         slice_mask[fill_mask == 1] = np.invert(slice_mask[fill_mask == 1])
     return slice_mask
 
 
-def get_contour_fill_mask(series_slice: Dataset, contour_coords):
+def get_contour_fill_mask(series_slice: Dataset, contour_coords, transformation_matrix: np.ndarray):
     # Format data
     reshaped_contour_data = np.reshape(contour_coords, [len(contour_coords) // 3, 3])
-    translated_contour_data  = translate_contour_to_pixel_coordinants(reshaped_contour_data, series_slice)
-    polygon = [np.array([translated_contour_data[:, :2]], dtype=np.int32)]
+    translated_contour_data = apply_transformation_to_3d_points(reshaped_contour_data, transformation_matrix)
+    polygon = [np.around([translated_contour_data[:, :2]]).astype(np.int32)]
```

Comment:
This changes test IOUs from 
0.95, 0.87, 0.85, 0.95
to
1.0, 0.96, 0.97, 1.0

Reply:
Amazing!

Reply:
Would you mind increasing their thresholds with the tests to reflect this?